src/favouritesPage/components/favouriteCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { Button, Card, LoadingSpinner } from 'hds-react';
+import { Row, Col } from 'react-grid-system';
+import { PlotSearch, PlotSearchTarget } from '../../plotSearch/types';
+import { IconCrossCircle, IconArrowDown, IconArrowUp } from 'hds-react';
```

Comment:
Not a huge deal, but imports could be split into library imports first and local imports after that whenever spotted.